src/main/java/com/microsoft/jenkins/appservice/commands/FileDeployCommand.java

```diff
@@ -25,6 +26,7 @@
 import java.util.List;
 
 public class FileDeployCommand implements ICommand<FileDeployCommand.IFileDeployCommandData> {
+    public static final int DEFAULT_MAX_RETRY_TIMES = 3;
```

Comment:
is this configurable?

Reply:
no, not necessary to expose this to users